### DISCUSSION TITLE
nuclei context deadline exceeded

### DISCUSSION THREAD

**Discussion by mnd5756242** [projectdiscovery/nuclei#4542] [Q&A]
Execute the following methods in batches：
   
     e := poc.Executer
     ctx := scan.NewScanContext(contextargs.NewWithInput(target))
     results, err = e.ExecuteWithResults(ctx)

It is normal to start execution, and then the output will be ：reported an error：**context deadline exceeded**
**The version used is 3.0**
What is the reason

**Comment by olearycrew** [comment]
This appears to be a duplicate of https://github.com/projectdiscovery/nuclei/discussions/4535 yes?

**Reply by mnd5756242** [reply]
yes , Do you know how to solve it

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
